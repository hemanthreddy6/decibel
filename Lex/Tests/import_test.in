improt myfile.bd ;
import 'hello.c'
import [file1.java] $
import 'girgr` :
import "a.c" & <bits/stdc++.h>
