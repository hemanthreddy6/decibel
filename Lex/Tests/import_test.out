IDENTIFIER
IDENTIFIER
INVALID_SYMBOL
IDENTIFIER
;
IMPORT
STRING_LITERAL
IMPORT
[
IDENTIFIER
INVALID_SYMBOL
IDENTIFIER
]
INVALID_SYMBOL
IMPORT
INVALID_SYMBOL
IDENTIFIER
INVALID_SYMBOL
:
IMPORT
STRING_LITERAL
&
<
IDENTIFIER
/
IDENTIFIER
+
+
INVALID_SYMBOL
IDENTIFIER
>
